mindee/client.py

```diff
@@ -74,6 +74,8 @@ def parse(
         :param cropper: Whether to include cropper results for each page.
             This performs a cropping operation on the server and will increase response time.
         """
+        if self.input_doc is None:
+            raise RuntimeError("The 'parse' function requires an input document.")
```

Comment:
Maybe use `TypeError` instead?

Because `None` is of type `NoneType`, which is of a different type than `XxxInputSource`...

https://docs.python.org/3/library/exceptions.html#TypeError